packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -349,6 +373,40 @@ export default class SrDisks extends Component {
     },
   ]
 
+  _getVdisByBaseCopy = createSelector(
+    () => this.props.vdis,
+    () => this.props.unmanagedVdis,
+    (vdis, unmanagedVdis) => {
+      const vdisByBaseCopy = {}
+
+      vdis.forEach(activeVdi => {
+        if (activeVdi.parent === undefined) {
+          return
+        }
+
+        let baseCopy = unmanagedVdis[activeVdi.parent]
+        let iterate = true
+        while (iterate) {
```

Comment:
Why don't you put the condition in the `while` directly?

Reply:
In case of a `activeVdi` don't have parent, it's mean this `VDI` is not in a `VDI chaine`, so we dont need to browse the `chaine` with the `while`.